tests/providers/google/cloud/operators/test_gcs_to_bigquery.py

```diff
@@ -8,7 +8,7 @@
 # with the License.  You may obtain a copy of the License at
 #
 #   http://www.apache.org/licenses/LICENSE-2.0
-#
+
```

Comment:
@joppevos -> this is the problem you have with the licence

Reply:
@potiuk My thanks. should have checked the file differences pre and post